happypanda/gallerydb.py

```diff
@@ -27,15 +27,15 @@
 
 from PyQt5.QtCore import QObject, pyqtSignal, QTime
 
-from utils import (today, ArchiveFile, generate_img_hash, delete_path,
+from happypanda.utils import (today, ArchiveFile, generate_img_hash, delete_path,
```

Comment:
Not doing the PEP8 changes yet, next PR. I just wanted all the imports updated with as few changes as possible. (Otherwise me reordering the imports last PR should have been ok)